solr/core/src/java/org/apache/solr/core/snapshots/SolrSnapshotsTool.java

```diff
@@ -227,12 +227,12 @@ public void describeSnapshot(String collectionName, String snapshotName) {
       String indexDirPath = coreSnap.getIndexDirPath();
       if (pathPrefix != null) {
         // If the path prefix is specified, rebuild the path to the index directory.
-        indexDirPath = new Path(pathPrefix, coreSnap.getIndexDirPath()).toString();
+        indexDirPath = Path.of(pathPrefix, coreSnap.getIndexDirPath()).toString();
       }
 
       List<String> paths = new ArrayList<>();
       for (String fileName : coreSnap.getFiles()) {
-        Path p = new Path(indexDirPath, fileName);
+        Path p = Path.of(indexDirPath, fileName);
```

Comment:
I looked at this closer and think I'm going to move `SolrSnapshotsTool` to the hdfs module. This change neuters the hdfs part which is pretty core to this tool. The move will at least keep it around and working - since all the other HDFS implementation is in the hdfs module.

Reply:
Addressed in 68e28ef1384070771b650329a5262dad5ac770af